src/include/ipc/compilator_info.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
s/compilator/compiler

Shouldn't we place the header inside src/include/sof ?

Reply:
Yes, src/include/sof will be better place. Done